relayer-engine/src/config/index.ts

```diff
@@ -38,6 +38,7 @@ export enum Mode {
 export type NodeURI = string;
 
 export interface CommonEnv {
+  namespace?: string;
```

Comment:
You also need to pipe this new config value through validation here https://github.com/wormhole-foundation/relayer-engine/blob/main/relayer-engine/src/config/validateConfig.ts#L25

Reply:
I did a pass-through from the raw value, but if you think we must enforce something, let me know, the only thing that I have in mind is any restriction over redis queue names, and in a quick search I did not find restriction on redis queues/keys